CMakeLists.txt

```diff
@@ -722,8 +732,16 @@ if(NOT SKIP_INSTALL_LIBRARIES AND NOT SKIP_INSTALL_ALL)
     include(CMakePackageConfigHelpers)
     write_basic_package_version_file(${CMAKE_CURRENT_BINARY_DIR}/minizip-config-version.cmake
         COMPATIBILITY SameMajorVersion)
-
-    file(WRITE minizip-config.cmake.in "@PACKAGE_INIT@")
+    set(MINIZIP_CONFIG_CONTENT "@PACKAGE_INIT@")
+    if(MINIZIP_DEP_PKG)
+        string(APPEND MINIZIP_CONFIG_CONTENT "\ninclude(CMakeFindDependencyMacro)")
+        foreach(PKG_NAME ${MINIZIP_DEP_PKG})
+            message("asdasdasd ${PKG_NAME}")
```

Comment:
I think some debugging info was left in.

Reply:
oh, sorry about that